README.md

```diff
@@ -1,42 +1,80 @@
 
-So what happens if you get it wrong? Nothing bad! We'll point you towards a more appropriate venue if necessary and, when possible, help migrate the issue for you. And please don't assume that a request to continue discussion of your issue elsewhere is a criticism, we genuinely want to ensure that your idea is reaching the right audience!
+Once two or three people from separate organizations have volunteered to be feature advocates then you should reach out to the CG chair and ask that a feature repo be created.
 
-## Code of Conduct
-**TL;DR: Treat other participants with respect or get banned.**
+## The CG chair made a new feature repo. Now what?
+
+One goal of the repo is to work through the nitty-gritty details of a feature with enough detail that it's clearly possible to implement and it's attractive to the relevant Working Group.
+
+The other (perhaps more important) goal of the repo is to attract the attention of implementers and topic experts and then find consensus for the feature details.
+
+Feature advocates (mentioned in the paragraphs above) are on the hook to do both technical and social work. It can take **a lot** of effort to gather enough feedback from a wide enough range of people that your feature will work for the entire immersive web.
+
+You start by writing an explainer document that lays out the context for the feature, captures the main discussion points from the proposals Issue, and runs through a few possible shapes that the feature's API or other technical artifacts could take. The goal of this document is to give readers the information they need to decide whether they want to work on the feature.
+
+Again, if you get stuck on this step (either in the writing or in the social aspects) then reach out to a community group chair. 
+
+## Our feature repo has all of the details need to implement a feature. Now what?
+
+Excellent! You've talked to enough of the community and worked together as feature advocates to nail down pretty much everything that one would need to know in order to implement a feature.
+
+If you've been prototyping the feature during the process of nailing down the design, you're probably in good shape. If not, you'll need to find someone to write code to prove out the design so that you can make the case that the feature is definitely ready to use in production.
+
+Ideally, you'll have prototypes that work on different platforms so it's clear that the feature is not a solution for just one vendor.
+
+During this process you should reach out to a CG chair and start the work of finding or creating a W3C working group that is a good fit for the feature.
```

Comment:
Here you say "a CG chair", elsewhere you say "the CG chair".  I'd just make it consistent.